addon/components/plugins/headings/headings-menu.ts

```diff
@@ -0,0 +1,67 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { NodeType, ProseController } from '@lblod/ember-rdfa-editor';
+import { setBlockType } from '@lblod/ember-rdfa-editor/commands/set-block-type';
+
+type Args = {
+  controller: ProseController;
+};
+export default class HeadingsMenu extends Component<Args> {
```

Comment:
The menu is janky when used in contexts where headings are not allowed, such as lists. Better to remove the menu when it is not applicable to be consistent with the other menus